docs/patterns/index.md

```diff
@@ -13,42 +14,72 @@ summaries:
   sticky-banner: Anchors an offer to the bottom edge of a page
   sticky-card: Anchors an offer to the right edge of a page
   video-thumbnail: Overlays a button that indicates video playback
+order: 0
+tags:
+  - pattern
+importElements: 
+  - rh-tile
 ---
 
+<link rel="stylesheet" href="{{ '/assets/packages/@rhds/elements/elements/rh-tile/rh-tile-lightdom.css' | url }}">
+
+<style>
+  .page-patterns .container .grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: var(--rh-space--2xl, 32px);
+    margin-block: var(--rh-space--2xl, 32px);
+  }
+
+  @container container (min-width: 567px) {
+    .page-patterns .container .grid {
+      grid-template-columns: 1fr 1fr;
+      gap: var(--rh-space--3xl, 48px);
+    }
+  }
+
+  @container container (min-width: 768px) {
+    .page-patterns .container .grid {
+      grid-template-columns: 1fr 1fr 1fr;
+    }
+  }   
+</style>
+
 {# NOTE: all images in this view need to be 340 by 200 px in order to maintain same ratio. #}
 
 ## Overview
 Patterns compose elements and tokens with content and validation rules to 
 create uniform, accessible experiences.
 
-<div class="multi-column--min-400-wide margin-top--10">
+<nav class="grid">
```

Comment:
Since there can be more that one <nav> on a page, the need unique `aria-label` elements. I suggest `<nav class="grid" aria-label="Patterns">`